src/demo/containers/App.tsx

```diff
@@ -170,13 +169,12 @@ export default class App extends React.Component<{}, AppState> {
                 </Link>
               </li>
             </ul>
-            <Route exact={true} path="/" component={Demo} />
-            <Route
-              path="/Playground"
-              component={(props: RouteChildrenProps) => (
-                <Playground {...props} token={token} />
-              )}
-            />
+            <Route exact={true} path="/">
+              <Demo />
+            </Route>
+            <Route path="/Playground">
+              <Playground />
```

Comment:
If the Playground caused too much trouble, maybe we should delete it from the codebase?